projects/packages/backup/src/js/components/Admin/index.js

```diff
@@ -117,40 +115,38 @@ const Admin = () => {
 	);
 };
 
-// Renders additional segments under the jp-hero area condition on having a backup plan
-const BackupSegments = ( hasBackupPlan, connectionLoaded ) => {
+// Render additional segments for the backup admin page under the jp-hero section.
+// If the user has a backup plan and is connected, we render the storage space segment.
+const BackupSegments = ( { hasBackupPlan, connectionLoaded } ) => {
```

Comment:
We were receiving the `hasBackupPlan` and `connectionLoaded` incorrectly. They are actually being [passed as props](https://github.com/Automattic/jetpack/blob/1a09629e9e50831d0bef63487c1529fd1b64decf/projects/packages/backup/src/js/components/Admin/index.js#L108).